pkg/operator/k8sutil/customresource.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
```

Comment:
```suggestion
Copyright 2020 The Rook Authors. All rights reserved.
```

Or did you move this file?